docs/dev/reference/config.md

```diff
@@ -106,18 +106,10 @@ contao:
                 formats:
 
                     # Examples (for webp settings analogous to the options in the backend):
-                    png:
-                        - webp
-                        - png
-                    jpg:
-                        - webp
-                        - jpg
-                    jpeg:
-                        - webp
-                        - jpeg
-                    gif:
-                        - webp
-                        - gif
+                    png: [webp, png]
+                    gif: [webp, gif]
+                    jpg: [webp, jpg]
+                    jpeg: [webp, jpeg]
```

Comment:
This will probably be reverted in the future, as this part of the documentation is just the output from `config:dump-reference`.